### PR TITLE
Bugfix/sns encrypt cmk

### PR DESCRIPTION
**Pull request by hernanku**

### CODE REVIEW

**Review by jamescrowley321 — APPROVED** [review]

**Review by jamescrowley321 — CHANGES_REQUESTED** [review]
Needs approval from @mayank0202 @tsaucier-sf @vijay-stephen

File: example/main.tf [hunk 1]
```diff
@@ -16,28 +16,29 @@ provider "aws" {
 }
 
 module "cloud_security" {
-  # source      = "git::https://github.com/sourcefuse/terraform-aws-arc-security.git?ref=feature/working-example"
-  source      = "../" // local development
+  # source = "git::https://github.com/sourcefuse/terraform-aws-arc-security.git?ref=bugfix/sns-encrypt-cmk"
```
Comment (tsaucier-sf) [inline example/main.tf:19]: Remove this

File: locals.tf [hunk 2]
```diff
@@ -46,4 +53,41 @@ locals {
     }
   }
 
+  kms_key_administrators = [
+    data.aws_iam_session_context.current.issuer_arn,
+    "arn:aws:iam::${data.aws_caller_identity.current.account_id}:root"
+  ]
+
+  key_policy = jsonencode({
+    Id      = "key-policy-1",
```
Comment (tsaucier-sf) [inline locals.tf:62]: We should let AWS auto-assign the Id, please omit.

**Comment by mayank0202** [comment]
THIS SHOULD BE CLOSED @jamescrowley321  @vijay-stephen  @hernanku

**Pull request closed without merging**